google/cloud/firestore_v1beta1/document.py

```diff
@@ -567,8 +570,11 @@ def __eq__(self, other):
         return self._reference == other._reference and self._data == other._data
 
     def __hash__(self):
-        seconds = self.update_time.seconds
-        nanos = self.update_time.nanos
+        # TODO(microgen): maybe add datetime_with_nanos to protoplus, revisit
+        # seconds = self.update_time.seconds
+        # nanos = self.update_time.nanos
+        seconds = int(self.update_time.timestamp())
+        nanos = 0
         return hash(self._reference) + hash(seconds) + hash(nanos)
```

Comment:
Same comment as earlier about precision.

Reply:
bug opened for protoplus, this should be done.